examples/eth.ts

```diff
@@ -0,0 +1,56 @@
+import { Kiln } from "../src/kiln";
+import type { Integration } from "../lib/types/integrations";
+import fs from "node:fs";
+import 'dotenv/config'
+
+
+const apiSecret = fs.readFileSync(`${__dirname}/fireblocks_secret.key`, 'utf8');
+
+const f = async () => {
```

Comment:
fyi latest version of node + bun can run async code directly

Reply:
oh sweet, I've updated examples